src/psyclone/domain/lfric/transformations/lfric_alg_invoke_2_psy_call_trans.py

```diff
@@ -80,6 +396,10 @@ def apply(self, node, options=None):
         :py:class:`psyclone.domain.lfric.algorithm.LFRicAlgorithmInvokeCall`
         :param options: a dictionary with options for transformations.
         :type options: Optional[Dict[str, Any]]
```

Comment:
In this case it's not optional?

Reply:
Again, it depends on your perspective. The argument is optional. I'm happy to change this and other cases if you disagree.

Reply:
It's a moot point and I don't care that much so we can leave it.